src/features/party/PartiesProvider.tsx

```diff
@@ -86,26 +87,26 @@ const { Provider: PartiesProvider, useCtx: usePartiesAllowedToInstantiateCtx } =
   }),
 );
 
-interface CurrentParty {
+interface SelectedParty {
   party: IParty | undefined;
-  currentIsValid: boolean | undefined;
+  selectedIsValid: boolean | undefined;
   userHasSelectedParty: boolean | undefined;
   setUserHasSelectedParty: (hasSelected: boolean) => void;
   setParty: (party: IParty) => Promise<IParty | undefined>;
 }
 
-const { Provider: RealCurrentPartyProvider, useCtx: useCurrentPartyCtx } = createContext<CurrentParty>({
-  name: 'CurrentParty',
+const { Provider: RealCSelectedPartyProvider, useCtx: useSelectedPartyCtx } = createContext<SelectedParty>({
+  name: 'SelectedParty',
```

Comment:
Was the `C` in `RealCSelectedPartyProvider` intentional?

Reply:
Nope 😅